### ISSUE TITLE
WCAG 3 Draft review: Conformance - How well will it work for our organization

### ISSUE THREAD

**Issue by maryjom** [w3c/silver#602]
### Questions 2 and 4 in the Introduction
- How well will this proposal of conformance work in your type of organization and why? 
- How will the change to a different conformance model than WCAG 2 affect your organization? How readily would you see moving from WCAG 2 to WCAG 3 when it is complete?

**Answer:** I am happy to see that WCAG 3 is working to incorporate methods to allow for flexibility in conformance, plus ways for organizations that go the extra mile to be recognized. However, the exact methods proposed in combination with each other appear to make the overall conformance assessment more complicated. In its unfinished form it is a little difficult to definitively assess the impact at this point but I thought it might be helpful to share some thoughts on how major changes like the conformance model in conjunction with new and changed individual requirements would potentially roll out at a large scale.

A company that has a very large number of different products as well as internally and externally facing web content, would have to make a number of adjustments which would take quite some time. Changes would need to be made:

1. to guidance
    - to teach the new terminology
    - to incorporate new and changed requirements
    - to assess the impact to our products and communicate that to the company
2. to test tools 
    - modify test rules for changed requirements in existing criteria 
    - splitting up tests when requirements are split to be more granular (I do like this idea though, because several current SC cover a lot)
    - add test rules for new requirements
    - provide ways to potentially automate path testing 
    - incorporate ways for test tools to count instances of the objects that apply to a requirement and how many pass or fail - and combine those that use different methods to pass.
3. to test strategies (to add focus on task flow) which include
    - how to document and test the task flow
    - how to incorporate results of testing the task flow into meeting an individual requirement, and into the overall conformance
4. to aggregation methodologies 
    - to choose between (or potentially aggregate) whole-page results vs. task flow results vs "view" results per requirement to report in an Accessibility Conformance Report (for products) or an accessibility statement (for websites) 
    - to aggregate per functional category to ensure 95% coverage in each to determine if bronze is met (Which, IMO is potentially unachievable in and of itself if there are a number of new/changed criteria introduced.)
    - to provide ways to report any additional user flow testing, associate with the various requirements, and aggregate to determine if silver or gold level is met
    - to ensure this is applied the same by all teams, tools will need to be devised to automate as much as possible.
5. to our reporting tools and strategies 
    - There will be added complexity (for some unknown length of time until regulations take up WCAG 3 fully) where reporting to WCAG 3 vs. current worldwide regulations will be pretty difficult. Since WCAG 3 is not backward compatible. I understand the reasons why it is not, but want to acknowledge that required regulatory reporting of accessibility requirements to existing worldwide policies and their adopted standards will become more complex for an unknown period of time until regulations begin to use the new standard. We will have to assess and potentially develop strategies to be able to continue to respond to regulations that require WCAG 2.x.

All-in-all, the proposed conformance model seems pretty complex. We are finding even the existing WCAG conformance is a lot to expect of teams working in an agile development process. They are delivering new content and features on a quick, regular schedule. Without some significant tooling available to them to make it easy to accomplish, there will be a lot of pushback to additional complexity.

I don't want to sound overly negative. We all know this is a very tricky line to walk. We all want to improve the user experience and make the web a place where as many users as possible can benefit from it. We do not want to create a conformance model where people will simply game it, making it meaningless. But, conformance has to have a level of simplicity that organizations can easily communicate what teams need to do and then teams can easily learn, carry out, and report on in a meaningful way. Consumers and procurers also need to be able to understand what the levels of conformance means for them.

**Comment by cwadamsoforacle** [comment]
Proposed response:  

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).

**Comment by maryjom** [comment]
Agree, so closing this issue.
